helpdesk_mgmt_fieldservice/views/res_partner.xml

```diff
@@ -0,0 +1,18 @@
+<odoo>
+
+    <record id="res_partner_form_ticket_context" model="ir.ui.view">
+        <field name="name">res.partner.form.ticket.context</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="service_location_id" invisible="1" />
+            <button name="action_view_helpdesk_tickets" position="attributes">
+                <attribute name="groups">fieldservice.group_fsm_user_own</attribute>
+                <attribute
+                    name="context"
+                >{'default_partner_id': id, 'default_fsm_location_id': service_location_id}</attribute>
+            </button>
+        </field>
+    </record>
```

Comment:
**issue:** it seems that in 14.0, this customization was made available only for users with the group `fieldservice.group_fsm_user_own`, while now you are making the whole button available only for the users with that group.

Could you please check and fix the issue to make the functionality the same as it was before?

Reply:
Removed groups_id and for accessing  service_location_id field which can be accessed with groups "fieldservice.group_fsm_user_own" added the field with invisible in the parent view

Reply:
Could you please double check? I feel like that to preserve the full functionality you should:
- Use two buttons, the one already existing and create a new one
- The first should be visible if you have the group, the other one not
- The one visible if you have the group should also have the customization made in this inheritance 

Let me know if everything is clear :)

Reply:
This seems solved.